code/game/objects/items/devices/scanners.dm

```diff
@@ -281,28 +281,30 @@ MASS SPECTROMETER
 		environment.garbage_collect()
 
 		if(abs(n2_concentration - N2STANDARD) < 20)
-			user << "<span class='info'>Nitrogen: [round(n2_concentration*100)] %</span>"
+			user << "<span class='info'>Nitrogen: [round(n2_concentration*100, 0.01)] %</span>"

```

Comment:
did you mean these to be 0.01 or 0.001? 

would be nice if you made these into a define, either way.